packages/extensions/shopify-cli-extensions/packages/shopify-cli-extensions/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "outDir": "./",
     "declaration": true,
     "strict": true,
+    "skipLibCheck": true,
```

Comment:
Why did we need to add this? If it's due to a typescript version conflict can we bump the TS version for this package?

Reply:
Correct. There're are two versions of the same nested dependency through different branches and the Typescript validation ends up failing because of that.